book/listings/list_widgets/6/main.rs

```diff
@@ -27,17 +27,12 @@ fn build_ui(app: &Application) {
         .build();
 
     // ANCHOR: string_list
-    // Create a `Vec<String` with number from 0 to 100_000
-    let vector: Vec<String> = (0..=100_000)
+    // Create a `StringList` with number from 0 to 100_000
+    // `StringList` implements FromIterator<String>
+    let model: StringList = (0..=100_000)
         .into_iter()
         .map(|number| format!("Task number {}", number))
         .collect();
```

Comment:
The usual pattern is `let foo = ....collect::<Type>();`. You generally use the turbofish instead of type ascriptions.

Reply:
Do you happen to have references for that?
In that case quite a few places have to be changed in the book I think.

Reply:
Yes, but I think it's also what's considered more idiomatic in Rust in general